README.md

```diff
@@ -38,6 +38,12 @@ This allows easy execution as e.g. `<Exec Command="$(InnoSetupCompiler) setup.is
 How to build it
 ---------------
 
+To update the build with GitHub Action on the package on the GitHub project:
+
+1. Change version in `.\.github\workflows\build-package.yml`
+2. Change version in `.\Tools.InnoSetup.nuspec`
+
+
```

Comment:
I am unsure of this being appropriate (certainly not as part of "version bump" commit, though). Here's why: having to look at commit history to see how bumping is done is a small obstacle to new contributors, yes, but only a small one. Having a checklist, on the other hand, tends to push people towards mindlessly following it.

If you feel like these instructions would be helpful, OK, you're certainly more competent to answer that than me because you *are* the target audience, but then it should IMHO be a subsection of "How to build it" appended to its end (building comes first, bumping version after that) and should included step 3. Carefully review Inno Setup release notes for changes that may impact the build script and/or nuspec file.

Reply:
I removed it.